src/ensembl/production/datacheck/tests/test_datacheck_app.py

```diff
@@ -0,0 +1,126 @@
+# .. See the NOTICE file distributed with this work for additional information
+#     regarding copyright ownership.
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#         http://www.apache.org/licenses/LICENSE-2.0
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+import unittest
+from flask import Flask, render_template, jsonify, Request, request
+from werkzeug.test import EnvironBuilder
+from ensembl.production.datacheck.forms import DatacheckSubmissionForm
+
+valid_payload ={
+                'server-server_name':'mysql://ensro@mysql-ens-sta-1:4512/' ,
```

Comment:
I think it best not to have proper server details here.

Reply:
agreed